src/routes/wallet/Send.svelte

```diff
@@ -23,28 +32,43 @@
 		TransferArgs,
 		TransferArg
 	} from '$lib/../declarations/icp_ledger/icp_ledger.did';
-	import type { _SERVICE as icrcLedgerInterface } from '$lib/../declarations/icrc_ledger/icrc_ledger.did';
+	import type {
+		Account,
+		_SERVICE as icrcLedgerInterface
+	} from '$lib/../declarations/icrc_ledger/icrc_ledger.did';
 	import type { _SERVICE as icpLedgerInterface } from '$lib/../declarations/icp_ledger/icp_ledger.did';
 	import { fade } from 'svelte/transition';
 	import Toast from '../Toast.svelte';
+	import { IcrcLedgerCanister } from '@dfinity/ledger-icrc';
 
 	let principal: string;
 	let isSending = false;
 	let dialog: HTMLDialogElement;
 
-	function isValidAmount(amount: BigNumber): boolean {
-		if (amount && $user) {
+	function isValidAmount(amount: BigNumber): boolean | undefined {
+		if ($user?.account === 'main') {
 			return (
-				$user.getBalance($selectedAsset.type).isGreaterThanOrEqualTo(amount) &&
+				$user?.getBalance($selectedAsset).isGreaterThanOrEqualTo(amount) &&
 				amount.isGreaterThanOrEqualTo(BigNumber(1).dividedBy(E8S))
 			);
 		} else {
-			return true;
+			return (
+				$ledgerDevice?.getBalance($selectedAsset).isGreaterThanOrEqualTo(amount) &&
+				amount.isGreaterThanOrEqualTo(BigNumber(1).dividedBy(E8S))
+			);
```

Comment:
?

Reply:
The selected amount you send has to be lower than you funds and greater than the 8th decimal.